File: src/test/resources/projects/kotlin-deps/KotlinLibrary.java
///usr/bin/env jbang
//DEPS com.openai:openai-java:4.52.0

// OpenAIClient is compiled from Kotlin and carries kotlin.Metadata.
// It verifies K2 can expose Kotlin-library light classes to Java JBang scripts.
import com.openai.client.OpenAIClient;

class KotlinLibrary {
    public static void main(String[] args) {
        OpenAIClient client = null;
        System.out.println("Kotlin dependency resolves: " + client);
    }
}

File: src/test/resources/projects/kotlin-deps/KotlinLibrary.kt
#!/usr/bin/env jbang
//DEPS com.openai:openai-java:4.52.0

// The same Kotlin-compiled dependency, consumed from a Kotlin JBang script.
import com.openai.client.OpenAIClient

fun main() {
    val client: OpenAIClient? = null
    println("Kotlin dependency resolves: $client")
}
